lambdas/handlers/create_document_reference_handler.py

```diff
@@ -152,6 +156,12 @@ def lambda_handler(event, context):
             500, "An error occurred when creating document reference", "POST"
         ).create_api_gateway_response()
         return response
+    except (LGInvalidFilesException, LGFileTypeException, LGFileNameException) as e:
+        logger.error(e)
+        response = ApiGatewayResponse(
+            400, "One or more if the files is not valid", "POST"
```

Comment:
I feel this should be more descriptive to give the user the info they need to fix the problem and try again. Since we're catching several different types of exception here, can we instead have a different error message depending on which exception is raised? e.g. 

File names do not fit into the pattern XofX_Lloyd_George_Record_[Patient Name]_[nhs number]_[DD-MM-YYYY].pdf
File type is incorrect- please ensure all Lloyd George files are PDF

and so on

Reply:
That's fine if it's done in the UI, as long as the user isn't going to get a message saying "400, "One or more if the files is not valid" without telling them what they need to change to make it valid - if the file names are invalid I feel we should tell them why so they can fix it and try again

Additionally, this should be 'of', not 'if' - just seeing that now!